app/views/complaints/index.html.erb

```diff
@@ -1,6 +1,6 @@
 <h1>Complaints List</h1>
 <% if user_signed_in? %>
-  <div><%= current_user["name"]%>'s complaints</div>
+   <%= render 'complaints_data' %>
```

Comment:
[suggestion] paired with comment in the partial, this could be `<%= render (@compliants.empty? ? 'no_complaints' : 'complaints_table') %>` Or something with more lines if I'm the only one who actually likes the ternary operator

Reply:
no i LOVE ternary